css/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+    background-color: antiquewhite;
+}
+
+.center {
+    text-align: center;
+
+}
+
+form {
+    width: 50%;
+    margin: auto;
+}
+
+.sixty {
+    width: 60%;
+}
+
+.margin {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+select {
+    width: 60%;
+}
```

Comment:
I think it would be better to use `.sixty` class on `<select>` tag so there is no repeating CSS rules.